warehouse/accumulo-extensions/src/test/java/datawave/ingest/table/volumeChoosers/ShardedTableDateBasedTieredVolumeChooserTest.java

```diff
@@ -0,0 +1,294 @@
+    @Test
+    public void testNewVolumesNotInOptionsForNewData() {
+        Set<String> oldOnlyOptions = new HashSet<>();
+        oldOnlyOptions.add("oldData1");
+        oldOnlyOptions.add("oldData2");
+        
+        String newVolumes = "newData1,newData2,newData3";
+        String oldVolumes = "oldData1,oldData2";
+        long daysBack = 125L;
+        
+        Map<Long,String> tiers = new HashMap<>();
+        tiers.put(daysBack, newVolumes);
+        tiers.put(Long.MAX_VALUE, oldVolumes);
+        
+        String shardId = "30000202_123";
+        setupMock(tiers, shardId);
+        ShardedTableDateBasedTieredVolumeChooser chooser = new ShardedTableDateBasedTieredVolumeChooser();
+        String choice = chooser.choose(env, oldOnlyOptions);
+        assertTrue(choice.contains("newData"));
+        
+    }
+    
+    @Test
+    public void testOldVolumesNotInOptionsForOldData() {
+        Set<String> newOnlyOptions = new HashSet<>();
+        newOnlyOptions.add("newData1");
+        newOnlyOptions.add("newData2");
+        newOnlyOptions.add("newData3");
+        
+        String newVolumes = "newData1,newData2,newData3";
+        String oldVolumes = "oldData1,oldData2";
+        long daysBack = 125L;
+        
+        Map<Long,String> tiers = new HashMap<>();
+        tiers.put(daysBack, newVolumes);
+        tiers.put(Long.MAX_VALUE, oldVolumes);
+        
+        String shardId = "20000202_123";
+        setupMock(tiers, shardId);
+        ShardedTableDateBasedTieredVolumeChooser chooser = new ShardedTableDateBasedTieredVolumeChooser();
+        chooser.choose(env, options);
+    }
+    
+    @Test
+    public void testShardIdDoesntMatchDatePattern() {
+        String newVolumes = "newData1,newData2,newData3";
+        String oldVolumes = "oldData1,oldData2";
+        long daysBack = 125L;
+        
+        Map<Long,String> tiers = new HashMap<>();
+        tiers.put(daysBack, newVolumes);
+        tiers.put(Long.MAX_VALUE, oldVolumes);
+        
+        String shardId = "HeyThatsNotADate";
+        setupMock(tiers, shardId);
+        ShardedTableDateBasedTieredVolumeChooser chooser = new ShardedTableDateBasedTieredVolumeChooser();
+        String choice = chooser.choose(env, options);
+        assertTrue(options.contains(choice));
+        
+    }
+    
+    @Test(expected = IllegalStateException.class)
+    public void testNegativeDaysBack() {
+        String newVolumes = "newData1,newData2,newData3";
+        String oldVolumes = "oldData1,oldData2";
+        long daysBack = -125L;
+        
+        Map<Long,String> tiers = new HashMap<>();
+        tiers.put(daysBack, newVolumes);
+        tiers.put(Long.MAX_VALUE, oldVolumes);
+        
+        String shardId = "20200101_123";
+        setupMock(tiers, shardId);
+        ShardedTableDateBasedTieredVolumeChooser chooser = new ShardedTableDateBasedTieredVolumeChooser();
+        String choice = chooser.choose(env, options);
+        assertTrue(options.contains(choice));
```

Comment:
This assertion is unreachable if we are passing the test and invalid if we somehow don't throw the IllegalStateException.
Recommend we just do:
```
chooser.choose(env, options);
```

Reply:
Done